src/Orleans/Statistics/CounterStatistic.cs

```diff
@@ -141,7 +151,7 @@ private static CounterStatistic FindOrCreate_Impl(StatisticName name, bool useDe
             }
         }
 
-        static public bool Delete(string name)
+        public static bool Delete(string name)
```

Comment:
the same question here. Just a little confused about why you turn a lot of private method into public. A little explanation will help